### PR TITLE
#170947650 Configure Dotenv

### PR DESCRIPTION
**Pull request by Kagorora**
#### What does this PR do?

- This PR configures Dotenv

#### Description of Task to be completed?

- I should be able to know what to fill in dotenv file by referring to the env.example file that shows all the required variables and their value

#### How should this be manually tested?

- run `git clone https://github.com/andela/knights-bn-backend.git` to download the repo

- run `cd knights-bn-backend` to go in the project

- run `git checkout ch-config-dotenv-170947650`

- create `.env` file and populate it by referring the instructions found in `env.example`

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [https://www.pivotaltracker.com/story/show/170947650](https://www.pivotaltracker.com/story/show/170947650)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,14 +14,16 @@
     "@babel/core": "^7.8.4",
     "@babel/node": "^7.8.4",
     "@babel/preset-env": "^7.8.4",
+    "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "nodemon": "^2.0.2",
```
Comment (T2Wil) [inline package.json]: Hey @Kagorora , `nodemon` should be installed in the package.json

**Review by T2Wil — APPROVED** [review]
LGTM

**Comment by MuhanguziDavid** [comment]
Please, resolve merge conflicts.

File: src/index.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
+import dotenv from 'dotenv';
 import app from './app';
 
-const port = process.env.PORT || 4000;
+dotenv.config();
+const port = process.env.PORT || 5000;
```
Comment (MuhanguziDavid) [inline src/index.js:5]: Any reason for changing the port from 4000 to 5000?

Comment (Kagorora) [reply src/index.js:5]: Since I have used 4000 in .env that line means that if 4000 is taken 5000 will be then used

**Pull request merged**